L/LLVM/common.jl

```diff
@@ -17,6 +17,7 @@ const llvm_tags = Dict(
     v"14.0.6" => "5c82f5309b10fab0adf6a94969e0dddffdb3dbce", # julia-14.0.6-3
     v"15.0.7" => "2593167b92dd2d27849e8bc331db2072a9b4bd7f", # julia-15.0.7-10
     v"16.0.6" => "499f87882a4ba1837ec12a280478cf4cb0d2753d", # julia-16.0.6-2
+    v"17.0.6" => "ac82491b6d79605ed3636112e9c831c597676b0b", # julia-17.SOMETHING TODO: I have no idea what upstream version number this corresponds to
```

Comment:
This commit corresponds to the head of JuliaLang/llvm-project's `julia-release/17.x` branch.

Reply:
I know, but I don't know what _upstream_ version that corresponds to, it's definitely _not_ 17.0.6 since that was released last week, while last upstream commit in that branch is from 5 months ago.  We need to update the base branch to match upstream 17.0.6, but I can't do that now, I don't have a good connection for cloning entire llvm repository.

Reply:
ahh i see